apps/oxlint/src/lib.rs

```diff
@@ -1,3 +1,7 @@
+use std::{ffi::OsStr, io::BufWriter};
```

Comment:
```rs
pub use oxc_linter::{
    ExternalLinter, ExternalLinterLintFileCb, ExternalLinterLoadPluginCb, LintFileResult,
    PluginLoadResult,
};
```

This should be moved up to after `use std::{ffi::OsStr, io::BufWriter};`. As per "The order" listed in #12723.